src/js/components/LayerPanel/LossControls.js

```diff
@@ -51,7 +51,6 @@ export default class LossControls extends Component {
         $('#loss-slider').ionRangeSlider({
           type: 'double',
           values: lossOptions.map(option => option.label),
-          min_interval: 1,
```

Comment:
Let's confirm with Christina: I think this is correct and the user should be allowed to only see data from one year:  our TreeCoverLossLayer is inclusive-inclusive on its date def query!